src/components/Typing.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+
+function Typing () {
```

Comment:
I think this component should be called `TypingIndicator`

Reply:
Ok